.gitignore

```diff
@@ -200,4 +200,7 @@ starters/standard/.pnpm-store
 .cursor/rules/user
 
 **/*.tsbuildinfo
-**/worker-configuration.d.ts
\ No newline at end of file
+**/worker-configuration.d.ts
+
+# Webstorm
+.idea/
```

Comment:
I can remove this if needed. I didn't find any obvious place for IDEs to place this under